### PR TITLE
#2021-git/1 skeleton

### PR DESCRIPTION
**Pull request by sedgweeck**
- main
- add sub div multi function

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+int sub(int, int);
+int div(int, int);
+int multi(int, int);
+
+int main(int argc, char* argv[])
+{
+    int a, b, c;
+    char operator;
+
+    printf("\Please enter a expression");
```
Comment (sedgweeck) [inline main.c:13]: comment1

Comment (sedgweeck) [inline main.c:13]: comment1

File: main.c [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            break;
+        case '*':
+            c = multi(a,b);
+            break;
+        case '/':
+            c = div(a,b);
+            break;
+    }
+
+    printf("result is %d\n", c);
```
Comment (sedgweeck) [inline main.c:32]: comment2

File: main.c (hunk 1 above)
Comment (shin-dong-jin) [reply main.c]: comment1

File: main.c (hunk 2 above)
Comment (shin-dong-jin) [reply main.c]: comment2

**Review by shin-dong-jin — APPROVED** [review]
1

**Pull request merged**